src/pages/WalletProviders.tsx

```diff
@@ -0,0 +1,206 @@
+                <label className="WalletCard__item__label">
+                  <Icon.Assets classname="WalletProvidersIcon" />
+                  Supported assets
+                </label>
+                <div className="WalletCard__item__value">
+                  {assets?.join(", ")}
+                </div>
+              </div>
+            </div>
+          </div>
+        </Card>
+
+        {/* Enable/Disable wallet modal */}
+        <Modal visible={updateWalletModal.visible} onClose={handleCloseModal}>
+          <Modal.Heading>
+            Confirm turning {updateWalletModal.enabled ? "off" : "on"} wallet
+            provider
+          </Modal.Heading>
+          <form
+            onSubmit={(event) => {
+              event.preventDefault();
+              handleUpdateWallet(
+                updateWalletModal.walletId,
+                !updateWalletModal.enabled,
+              );
+            }}
+            onReset={handleCloseModal}
+          >
+            <Modal.Body>
+              <div className="Note">
+                Make sure the wallet provider knows you have added them as they
+                will have to add you in order for this to work.
+              </div>
+              <Notification
+                variant="warning"
+                title="This will allow ANY disbursement to this provider"
+              ></Notification>
+            </Modal.Body>
+            <Modal.Footer>
+              <Button size="sm" variant="secondary" type="reset">
+                Cancel
+              </Button>
+              <Button
+                size="sm"
+                variant={updateWalletModal.enabled ? "destructive" : "primary"}
+                type="submit"
+              >
+                Turn {updateWalletModal.enabled ? "off" : "on"}
+              </Button>
+            </Modal.Footer>
+          </form>
+        </Modal>
+      </>
+    );
+  };
+
+  return (
+    <>
+      <SectionHeader>
+        <SectionHeader.Row>
+          <SectionHeader.Content>
+            <Heading as="h2" size="sm">
+              Wallet Providers
+            </Heading>
+          </SectionHeader.Content>
+        </SectionHeader.Row>
+      </SectionHeader>
+
+      <div className="CardStack">
+        <Card>
+          <div className="CardStack__card">
+            <div className="CardStack__title">
+              <InfoTooltip infoText="">My Wallet Providers</InfoTooltip>
+            </div>
+
+            {myWallets?.map((item) =>
+              renderWalletCard(
+                item.name,
+                item.id,
+                item.homepage,
+                item.enabled,
+                item.assets?.map((e) => e.code),
+              ),
+            )}
+          </div>
+        </Card>
+
+        <Card>
+          <div className="CardStack__card">
+            <div className="CardStack__title">
+              <InfoTooltip infoText="">Available Wallet Providers</InfoTooltip>
```

Comment:
Missing info text here too

Reply:
@torisamples can you provide the info text?